spec/models/queue_item_spec.rb

```diff
@@ -43,4 +47,33 @@
       expect(queue_item.category).to eq video.category
     end
   end
+
+  describe "#update_rating" do
+    let(:user) { Fabricate(:user) }
+    let(:video) { Fabricate(:video) }
+    let(:queue_item) { Fabricate(:queue_item, user: user, video: video) }
+
+    context "review is not present" do
+      it "creates a new review" do
+        queue_item.rating = 5
+        expect(Review.count).to eq 1
+      end
+    end
+
+    context "review is present" do
+      before do
+        @review = Fabricate(:review, user: user, video: video)
+      end
+
+      it "changes the rating of the review" do
+        queue_item.rating = 5
+        expect(@review.reload.rating).to eq 5
+      end
+
+      it "clears the rating of the review" do
+        queue_item.rating = nil
+        expect(@review.reload.rating).to be_nil
```

Comment:
Rather than instance variables, I would suggest to use `let`.

Read more [here](http://www.betterspecs.org/#let).